ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/network/DefaultP2PNetwork.java

```diff
@@ -144,7 +146,7 @@ public class DefaultP2PNetwork implements P2PNetwork {
   private final AtomicBoolean started = new AtomicBoolean(false);
   private final AtomicBoolean stopped = new AtomicBoolean(false);
   private final CountDownLatch shutdownLatch = new CountDownLatch(2);
-  private final Duration shutdownTimeout = Duration.ofMinutes(1);
+  private final Duration shutdownTimeout = Duration.ofSeconds(15);
```

Comment:
what's the impact of this change?

Reply:
There is a timeout happening if the network cannot shutdown within that time range. If we have to wait more than 15 seconds for this to shutdown we might as well kill it.

Reply:
👍